measurements/api/measurements.py

```diff
@@ -213,20 +206,21 @@ def list_measurements(
     # boolean arguments, that is logically the same of applying no filtering at
     # all.
     if failure is not None:
-        if set(clean_list_arg(failure)) == set(['true', 'false']):
+        if set(failure) == set(['true', 'false']):
             failure = None
         else:
-            failure = failure == 'true'
+            failure = failure == ['true']
```

Comment:
That's probably `set(failure) == {'true'}` because of `&failure=true&failure=true`

Reply:
We don't currently support the multi variable style of passing arguments, but only the csv one (`true,false`). Currently if you pass `failure=true,true` you are right to point out that it will not actually be passed as true, but I would consider this a mis-usage of the API.

Maybe in future versions we can be more tollerant.